hail/python/hail/expr/functions.py

```diff
@@ -3711,7 +3711,23 @@ def empty_array(t: Union[HailType, str]) -> ArrayExpression:
     return construct_expr(ir, array_t)
 
 
-def _ndarray(collection, row_major=True):
+def _ndarray(collection, row_major=None):
+    """Construct a hail ndarray from either a `NumPy` ndarray or python value/nested lists.
+    If `row_major` is ``None`` and the input is a `NumPy` ndarray, the ndarray's existing ordering will
+    be used (column major if that's how it is stored and row major otherwise). Note the exact memory layout
+    is not preserved if it is not in one of the two orderings. If the input is not a `NumPy` ndarray,
```

Comment:
Do I understand this correctly that if I pass in a ndarray in column major order and set row_major to true, then Hail will store it in row_major=True order?

Reply:
yes, maybe there's a clearly way for me to phrase that it's a little confusing

Reply:
Add `...(column major if that's how it is stored and row major otherwise). Otherwise, Hail will store the array as specified by row_major.`